queries/orderbook/prod_batch_rewards.sql

```diff
@@ -304,7 +304,10 @@ order_protocol_fee_prices AS (
         END AS network_fee_correction,
         opf.sell_token as network_fee_token,
         ap_surplus.price / pow(10, 18) as surplus_token_native_price,
-        ap_protocol.price / pow(10, 18) as protocol_fee_token_native_price,
+        CASE
+            WHEN opf.order_uid = '\x942e55be89314c5e799a12b487387a1221e5671bd17222a1bc69cb2b7728a4eea53a13a80d72a855481de5211e7654fabdfe352666e32aec' then 1.9088477506249592e-5
```

Comment:
If the price was only wrong in this one auction, I would prefer a solution only changing the price for one transaction hash.

```suggestion
            WHEN opf.tx_hash = '\x75e9283a7083e159aab70ca4462d16bb66d7309f504106924bd5fe1ff885fe2f' then 1.9088477506249592e-5
```

This is not the same as restricting to an order uid since this is a partially fillable order with multiple fills in the accounting period.

Reply:
Good catch

Reply:
I addressed it by also filtering based on auction id